innopoints/views/file.py

```diff
@@ -1,4 +1,15 @@
+"""Views related to file management.
+
+StaticFile:
+- POST /file/{namespace}
+- GET /file/{file_id}
+- DELETE /file/{file_id}
+"""
+
+import logging
+
```

Comment:
extra empty line

Reply:
Nope, it's between built-ins and external packages

Reply:
But `mimetypes` and `requests` are built-in